sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/JsonToProtoValueConverter.java

```diff
@@ -0,0 +1,38 @@
+package io.stargate.sgv2.restsvc.grpc;
+
+import io.stargate.grpc.Values;
+import io.stargate.proto.QueryOuterClass;
+import java.math.BigInteger;
+
+public class JsonToProtoValueConverter {
+  public static QueryOuterClass.Value NULL =
+      QueryOuterClass.Value.newBuilder()
+          .setNull(QueryOuterClass.Value.Null.newBuilder().build())
+          .build();
+
+  private JsonToProtoValueConverter() {}
+
+  public static QueryOuterClass.Value protoValueFor(Object javaValue) {
+    if (javaValue == null) {
+      return Values.NULL;
+    }
+    if (javaValue instanceof String) {
+      return Values.of((String) javaValue);
+    }
+    if (javaValue instanceof Number) {
+      if (javaValue instanceof Long) {
+        return Values.of((Long) javaValue);
+      }
+      if (javaValue instanceof Integer) {
+        return Values.of((Integer) javaValue);
+      }
+      if (javaValue instanceof BigInteger) {
+        return Values.of((BigInteger) javaValue);
+      }
```

Comment:
what about `Float`?

Reply:
Yes, for sake of completeness I'll need to add `Double`, `Float` and `BigDecimal` too. thanks!